tests/lax_numpy_test.py

```diff
@@ -4855,20 +4855,62 @@ def testArangeJit(self):
     self.assertAllClose(ans, expected)
 
   @jtu.sample_product(
-    args=[(5,), (0, 5)],
+    args=[(5,), (0, 5), (1, 5)],
     specify_device=[True, False],
   )
   def testArangeJaxpr(self, args, specify_device):
     device = jax.devices()[-1] if specify_device else None
     kwargs = {"device": device}
     jaxpr = jax.make_jaxpr(lambda: jnp.arange(*args, **kwargs))()
-    # We have 2 statements in jaxpr:
-    # [a:i32[5] = iota[dimension=0 dtype=int32 shape=(5,)],
-    #  a:i32[5] = device_put[devices=[None] srcs=[None]] b]
-    num_eqs = 2 if device is not None else 1
+
+    # Determine expected number of equations
+    start = 0 if len(args) == 1 else args[0]
```

Comment:
Tests should have minimal logic. Rather than putting this logic in the test, we should write a separate test for the case where the start value is non-zero.